src/Servers/Kestrel/Core/test/HttpHeadersTests.cs

```diff
@@ -174,6 +174,9 @@ public void TestParseConnectionMultipleValues(string value1, string value2, int
         [InlineData(",, ", (int)(TransferCoding.None))]
         [InlineData(" , ,", (int)(TransferCoding.None))]
         [InlineData(" , , ", (int)(TransferCoding.None))]
+        [InlineData("c", (int)(TransferCoding.Other))]
+        [InlineData("z", (int)(TransferCoding.Other))]
+        [InlineData("chunk", (int)(TransferCoding.Other))]
         [InlineData("chunked,", (int)(TransferCoding.Chunked))]
```

Comment:
Should this really be considered `TransferCoding.Chunked` or should it be `Other` instead?

Reply:
Empty values are dropped as insignificant. This is the same as:
```
Transfer-Encoding: chunked
Transfer-Encoding:
```